### DISCUSSION TITLE
ETA for next release

### DISCUSSION THREAD

**Discussion by royNiladri** [devicons/devicon#2098] [General]
Hello maintainers! 
First of all thank you all for coming up with the project and actively maintaining it. I have been using it in my GitHub readme, as well as my personal site. But i see the last release was in Mar 2022, and a lot more has been contributed. It would benefit us and likely a lot more users if we can have another release soon with whatever have already been developed. I am using the image url mode. Thanks again!

**Comment by Snailedlt** [comment]
I hope we'll be able to make a release in January. We've been working on a release for A LONG TIME now, so hopefully this is all that's left: https://github.com/devicons/devicon/pull/1998#issuecomment-1807542194.

If you just want to use the icons now before it's released you can go to the `develop` branch and copy the image links from `/icons/<iconname>/<iconname-versionname>.svg`

**Reply by royNiladri** [reply]
Thank you!! Will use the develop branch raw icons for now! :)

**Reply by royNiladri** [reply]
For others looking at the same thread, the referred URL looks like this:
https://raw.githubusercontent.com/devicons/devicon/develop/icons/angular/angular-original.svg

**Reply by Snailedlt** [reply]
We only have 2 PR's left now btw, so if you, or the next person wants to review one (or both), that would be highly appreciated!

The PR's that need to be reviewed:
- [nextjs](https://github.com/devicons/devicon/pull/2094)
- [microsoftsqlserver](https://github.com/devicons/devicon/pull/2094)
